developer-docs-site/docs/guides/move-guides/mint-nft-cli.md

```diff
@@ -81,28 +193,120 @@ aptos move run --function-id 8cdf69c8c93fee36ed83f8882908060c1335ed39a827c08dbb5
 
 ## Find the NFT in your Petra wallet
 
-1. Run `more ~/.aptos/config.yaml` to see the `nft-receiver` private key and then copy it.
+1. Open the Petra Wallet Chrome extension.
 
-1. Install the [Petra Wallet](../../guides/install-petra-wallet.md) Chrome extension.
+2. Go to *Petra > Settings > Network > Testnet*.
 
-1. Select the [Testnet network](https://petra.app/docs/use) in the wallet via *Petra settings > Network > Testnet*.
+3. Click **Library** at bottom.
 
-1. Use your testnet coins to send a transaction and mint an NFT. Obtain more and connect your wallet to the faucet at: https://aptoslabs.com/testnet-faucet
+4. See the NFT in your wallet:
 
-1. Go to *Petra > Settings > Switch account > Add Account > Import private key*.
+  ![Wallet library NFT](../../../static/img/petra-nft-library.png "See the wallet library with NFT")
```

Comment:
unfortunately the NFT is only available for my own contract in the private tutorial repo (since I uploaded the image using bundlr), the folks following the tutorial here won't receive an actual NFT when they deploy their own contract. unless we want to have a designer to put out a simple picture for this tutorial (@movekevin do you think this would be worth doing? think it might be more fun if folks can get a testnet "certificate" that they complete this tutorial). I'd be happy to put the picture in the token uri and update the tutorial with the new link : )

Reply:
Awesome! @movekevin and @cathyaptos , what do you think of an Aptos Certificate of NFT Completion? :-)
Thanks @0xchloe for the great idea!

Reply:
Ping @movekevin to see if we can provide tutorial takers with some visible results of success.

Reply:
I have removed reference to this. It does leave a bit of a gap on visualizing success.